src/instrumenter/annotations.ts

```diff
@@ -203,82 +270,45 @@ export class AnnotationExtractor {
             throw e;
         }
 
-        const annotationLoc: OffsetRange = offsetBy(
-            offsetBy(rangeToOffsetRange(annotationLocRelToRegex), match.index),
-            meta.loc
-        );
-
-        const exprLoc = offsetBy(offsetBy(rangeToOffsetRange(exprRange), match.index), meta.loc);
-
-        if (!hasSemi) {
-            let scope: string;
-
-            if (meta.target instanceof ContractDefinition) {
-                scope = `contract ${meta.target.name}`;
-            } else {
-                const prefix =
-                    meta.target.vScope instanceof SourceUnit
-                        ? ""
-                        : (meta.target.vScope as ContractDefinition).name + ".";
-
-                scope =
-                    meta.target instanceof FunctionDefinition
-                        ? `function ${prefix}${meta.target.name}`
-                        : prefix + meta.target.name;
-            }
-
-            const errRange = rangeToLocRange(annotationLoc[0], annotationLoc[1], source);
-
-            throw new SyntaxError(
-                `Line ${errRange.start.line} of ${scope} documentation string looks like an annotation but is not terminated by a semicolon ";" and is ignored: ${annotationOrig}`,
+        if (parsedAnnot instanceof SProperty) {
```

Comment:
I think we can reduce nesting here via early `return`s.

Reply:
Fixed in 031fe1c.